stdpopsim/arabidopsis_thaliana.py

```diff
@@ -77,14 +77,41 @@ class Salome2012(genetic_maps.GeneticMap):
 default_generation_time = 1.0
 
 
-class Durvasula2017MSMC(models.Model):
+class ArabidopsisThalianaModel(models.Model):
+    """
+    TODO: documentation
+    """
+    def __init__(self):
+        super().__init__()
+        self.generation_time = default_generation_time
+        self.default_population_size = 1000
+
+
+class GenericConstantSize(ArabidopsisThalianaModel, generic_models.ConstantSizeMixin):
+    def __init__(self):
+        ArabidopsisThalianaModel.__init__(self)
+        generic_models.ConstantSizeMixin.__init__(self, self.default_population_size)
+
+
+class GenericTwoEpoch(ArabidopsisThalianaModel, generic_models.TwoEpochMixin):
+    def __init__(self, n2=None, t=None):
+        ArabidopsisThalianaModel.__init__(self)
+        n1 = self.default_population_size
```

Comment:
Two things I don't like about this: (a) there's a bunch of arbitrary decisions being made here and (b) we're duplicating these arbitrary decisions in different species.

I don't have a better suggestion right now though.